proto/regen/data/v1/types.proto

```diff
@@ -132,15 +132,51 @@ enum GraphCanonicalizationAlgorithm {
   // unspecified and invalid
   GRAPH_CANONICALIZATION_ALGORITHM_UNSPECIFIED = 0;
 
-  // URDNA2015 graph hashing
+  // URDNA2015 graph canonicalization algorithm.
   GRAPH_CANONICALIZATION_ALGORITHM_URDNA2015 = 1;
+
+  // RDFC 1.0 graph canonicalization algorithm. Essentially the same as URDNA2015 with some
+  // small clarifications around escaping of escape characters. New users should use this
+  // instead of URDNA2015.
+  GRAPH_CANONICALIZATION_ALGORITHM_RDFC_1_0 = 2;
 }
 
 // GraphMerkleTree is the graph merkle tree type used for hashing, if any
 enum GraphMerkleTree {
 
   // unspecified and valid
   GRAPH_MERKLE_TREE_NONE_UNSPECIFIED = 0;
+
+  // specifies that the content hash for the graph is based on the following merkle tree algorithm:
+  //
+  // 1. the graph is canonicalized using the specified canonicalization algorithm
+  // 2. the whole canonicalized graph is hashed using the specified digest algorithm and this
+  //    hash is used as the salt
+  // 3. each triple in the canonicalized graph is hashed as follows:
+  //    a. the subject is hashed using the specified digest algorithm and the salt prefix
+  //    b. the predicate is hashed using the specified digest algorithm and the salt prefix
+  //    c. the object is hashed using the specified digest algorithm and the salt prefix
+  //    e. the resulting hashes are concatenated and hashed using the specified digest algorithm
```

Comment:
We could add hash iterations in these steps to make this more resistant to brute forcing. The number of iterations could either be fixed or configurable, although if it is configurable the question is where this is stored? One option is to create a few variants of this enum such as  GRAPH_MERKLE_TREE_SIMPLE_PRIVACY_PRESERVING (no iterations),  GRAPH_MERKLE_TREE_SIMPLE_PRIVACY_PRESERVING_10000,  GRAPH_MERKLE_TREE_SIMPLE_PRIVACY_PRESERVING_100000, etc.